client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -147,6 +148,19 @@ export default React.createClass( {
 		return {};
 	},
 
+	getDefaultActionLinks( plugin ) {
+		let adminUrl = get( this.props, 'selectedSite.options.admin_url' );
+		const pluginSlug = get( plugin, 'slug' );
+
+		if ( pluginSlug === 'vaultpress' ) {
+			adminUrl += '/admin.php?page=vaultpress';
+		}
+
+		return adminUrl
+			? { [ i18n.translate( 'WP Admin' ) ]: adminUrl }
+			: null;
+	},
+
```

Comment:
Reading this method again makes me think this isn't going to scale well to other plugins. Isn't this something that should rather be fixed at plugin level itself? Or can we at least generalize this so it works for non-Vaultpress plugins, too?